sticker-utils/src/bin/sticker-dep2label.rs

```diff
@@ -1,60 +1,76 @@
-use std::env::args;
 use std::io::BufWriter;
 use std::process;
 
+use clap::{App, AppSettings, Arg};
 use conllx::graph::Node;
 use conllx::io::{ReadSentence, WriteSentence};
 use conllx::token::Features;
-use getopts::Options;
 use stdinout::{Input, OrExit, Output};
 
 use sticker::depparse::{RelativePOSEncoder, RelativePositionEncoder};
 use sticker::SentenceEncoder;
 
-fn print_usage(program: &str, opts: Options) {
-    let brief = format!("Usage: {} [options] [INPUT] [OUTPUT]", program);
-    print!("{}", opts.usage(&brief));
-    process::exit(1);
+static ENCODER: &str = "ENCODER";
+static INPUT: &str = "INPUT";
+static OUTPUT: &str = "OUTPUT";
+
+static DEFAULT_CLAP_SETTINGS: &[AppSettings] = &[
+    AppSettings::DontCollapseArgsInUsage,
+    AppSettings::UnifiedHelpMessage,
+];
+
+pub struct Dep2LabelApp {
+    encoder: String,
+    input: Option<String>,
+    output: Option<String>,
 }
 
-fn main() {
-    let args: Vec<String> = args().collect();
-    let program = args[0].clone();
-
-    let mut opts = Options::new();
-    opts.optopt(
-        "e",
-        "encoder",
-        "dependency encoder",
-        "rel_pos or rel_position (default: rel_pos)",
-    );
-    opts.optflag("h", "help", "print this help menu");
-    let matches = opts.parse(&args[1..]).or_exit("Cannot parse options", 1);
-
-    if matches.opt_present("h") {
-        print_usage(&program, opts);
-        return;
+impl Dep2LabelApp {
+    fn new() -> Self {
+        let matches = App::new("sticker-dep2label")
+            .settings(DEFAULT_CLAP_SETTINGS)
+            .arg(
+                Arg::with_name(ENCODER)
+                    .short("e")
+                    .long("encoder")
+                    .value_name("ENC")
+                    .help("Dependency encoder (rel_pos or rel_position, default: rel_pos)")
```

Comment:
add `.possible_values(&["rel_pos", "rel_position"])` and perhaps `.default_value("rel_pos")`?